Source/Core/VideoCommon/XFStructs.cpp

```diff
@@ -192,6 +194,7 @@ static void XFRegWritten(int transferSize, u32 baseAddress, DataReader src)
     case 0x1017:
 
     default:
+      DolphinAnalytics::Instance().ReportGameQuirk(GameQuirk::USES_UNKNOWN_XF_COMMAND);
```

Comment:
We want to report 0 here?  Any idea why it ignored for the warn?

Reply:
Looks like 636bedb207784db2d58b9986464f6a863677b59e / #3242 hid those logs, specifically because of warnings on FifoCI, but doesn't specify what game did it.  Assuming it was a single game, IMO it's still useful to track it.

Reply:
I've added c702180fe7e1a15421533fd1e477c5dfe1c79703 temporarily to get a list of these from FifoCI.

Reply:
Looks like the source of the problem is that the FIFO player copies all of the registers on startup, which includes the unused ones; this produces log messages for everything (but shouldn't affect normal playback).  1099475d461f859ebde6e01e871973d38b0d37bd is a hacky workaround where the unused ones just aren't copied, but I don't think it's a great implementation.

0x1007 in particular is odd, because YAGCD says it's `Perf1` (0x1006 is `Perf0` which Dolphin calls `XFMEM_SETGPMETRIC`); games don't seem to use it though.  It's marked as known in the actual XFMemory struct, but there is no constant for it.  And weirder still is this code (which seems very unlikely to be correct):

https://github.com/dolphin-emu/dolphin/blob/8d2b0fff8a066281365072f827321e47bfed8c72/Source/Core/VideoCommon/XFStructs.cpp#L41-L48

Reply:
Having looked at other parts of the code, it turns out `nextAddress` advances address forward, but also updates the count, so it's more of an optimisation that happens to be rather unnecessary in this case, but makes more sense for e.g. `XFMEM_SETPROJECTION` where usually all 7 registers are updated at once, but it would be bad to update `g_vertex_manager` et al 7 times for that.

My earlier hackfix was backwards (I was skipping everything *but* the unknown registers...); 7ea201cf1cedd305af7c3b6008b7e2e2eff9187a is a correct implementation.  Note that that produces [rendering differences](https://fifo.ci/version/ae0f6042606541a063df68227e63d7416b6f1601/) but it seems that those are only from timing changes (if instead of skipping unknown registers, the last valid register is repeated (2bd038e9d5852e0df6d62b2556365ce11f3c30d9), [there are no differences](https://fifo.ci/version/2bd038e9d5852e0df6d62b2556365ce11f3c30d9/)); since this also only happened on the first frame in the fifo player, I'm not going to worry about it.

With skipping unknown registers, it should be fine to log a warning even when it's set to 0; none of the testcases on fifo.ci have any unknown registers in the fifo itself.